### ISSUE TITLE
Possible issue with BLE return code processing

### ISSUE THREAD

**Issue by markoer** [eclipse/kura#1857]
The rest of the failure codes produce KuraBluetoothCommandException, except for this one. Is this intentional? If so, just close the issue, otherwise I can submit the update with a PR resulting from #1514.

>https://github.com/eclipse/kura/blob/42080d54aafc1f17be3fa124206bb3eb1987bde1/kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/beacon/BluetoothLeBeaconManagerImpl.java#L237

**Comment by pierantoniomerlino** [comment]
@markoer Thanks for the clarification.
From the official documentation the "0c" means "The Command Disallowed error code indicates that the command requested cannot be executed because the Controller is in a state where it cannot process this command at this time." This error code is usually used when a command is run twice. If the app calls a stop discovery command twice, the second call returns "0c". So, I preferred to not throw an exception, but only to report it on log.

Long story short, it is intentional.

**Comment by markoer** [comment]
Thanks for the info. I will add it to the code comment.
